utils/tools.address.ts

```diff
@@ -11,6 +11,21 @@ export type TAddressWagmi = `0x${string}`;
 export type TAddress = TAddressWagmi;
 export type TAddressLike = TAddressSmol | TAddressWagmi | string;
 
+export type TInputAddressLike = {
+	address: TAddress | undefined;
+	label: string;
+	isValid: boolean | 'undetermined';
+	source?: 'typed' | 'addressBook' | 'defaultValue';
+	error?: string;
+};
+
+export const defaultInputAddressLike: TInputAddressLike = {
+	address: undefined,
+	label: '',
+	isValid: 'undetermined',
+	source: 'typed'
+};
+
```

Comment:
This const was exported from the `SmolAddressInput` component to the context provider which is not good. Decided to move this to constants